### DISCUSSION TITLE
fuse unmounted by kernel due to download slow

### DISCUSSION THREAD

**Discussion by yzcwf** [juicedata/juicefs#4552] [General]
When the download speed is particularly slow, the passive unmounted operation is triggered. I suspect it is due to the fuse read timeout. It is acceptable for file reading to fail, but now that the fuse has been unmounted, all files cannot be read

<img width="728" alt="image" src="https://github.com/juicedata/juicefs/assets/30282746/35e02be7-acb7-4b08-bb3b-5da2b1671387">

**Comment by davies** [comment]
How can we reproduce the `passive unmounted operation`?

**Comment by yzcwf** [comment]
<img width="787" alt="image" src="https://github.com/juicedata/juicefs/assets/30282746/da61bf7e-ab49-45fe-b942-a266c9591c18">
only reproduce on macOS

**Comment by davies** [comment]
Then it's a feature from macFUSE, we don't have much to do.
